org.uqbar.project.wollok.ui/src/org/uqbar/project/wollok/ui/wizard/WollokDslNewProject.xpt

```diff
@@ -28,5 +28,9 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%5p [%t] (%F:%L) - %m%n
 «ENDFILE»
 
+«FILE "src/WOLLOK.ROOT"-»
```

Comment:
This is the new file that determines where the root code is for project-relative imports, right ?

In the future I think that it would be nice to model it in a file like a pom, including data as version and dependencies, etc.
Kind of a pom.xml or package.json.